public/404.html

```diff
@@ -1 +1,39 @@
-<html><head><meta http-equiv="refresh" content="0; URL=/"></head><body></body></html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Single Page Apps for GitHub Pages</title>
+    <script type="text/javascript">
+      // Single Page Apps for GitHub Pages
```

Comment:
Do we need to include this description? @pablo-mayrgundter

Reply:
Yeah, it was meant as a copy/paste block from source, so keeping it intact is a nice nod to the original dev.  Also, makes it easy to compare with original later.

Reply:
Ok, did some trimming.  Please take another look (first PTAL!).